### PR TITLE
EASY-2438: Convert to maven multi-module

### PR DESCRIPTION
**Pull request by pfrolke**
fixes EASY-2438

#### When applied it will
* split project into two submodules (lib, command)

#### Where should the reviewer @DANS-KNAW/easy start?
* inspect changes made to project structure and different pom.xml files.

#### How should this be manually tested?
* build the project from the root directory using `mvn clean install`

#### related pull requests on github
repo                       | PR
-------------------------- | -----------------
easy-archive-bag                      | [PR61](https://github.com/DANS-KNAW/easy-archive-bag/pull/61)

### CODE REVIEW

**Comment by pfrolke** [comment]
This one took me the most time since the classes here where more interconnected. I ended up splitting `EasyStageFileItem.scala` into `lib/.../EasyStageFileItem.scala` and `command/.../EasyStageFileItemCommand.scala` and `FileItemSettings.scala` into `lib/.../FileItemSettings.scala` and `command/.../FileItemApplySettings.scala`.
Reviewers should take extra care reviewing these classes and their respective tests.

**Comment by rvanheest** [comment]
please don't include the `.idea` directory in your commits

File: command/src/main/scala/nl.knaw.dans.easy.stage.command/Command.scala [hunk 1]
```diff
@@ -34,7 +36,7 @@ object Command extends App {
   implicit val settings: Settings = new Settings(
     ownerId = getUserId(clo.deposit()),
     submissionTimestamp = if (clo.submissionTimestamp.isSupplied) clo.submissionTimestamp()
-                          else DateTime.now,
+    else DateTime.now,
```
Comment (rvanheest) [inline command/src/main/scala/nl.knaw.dans.easy.stage.command/Command.scala]: if this formatting change was done by IntelliJ, it means you don't have the right formatter installed 😉
See https://github.com/DANS-KNAW/easy-module-archetype/blob/master/dans-intellij-codestyles.xml and https://github.com/DANS-KNAW/easy-module-archetype/blob/master/dans-intellij-inspections.xml

Comment (rvanheest) [reply command/src/main/scala/nl.knaw.dans.easy.stage.command/Command.scala]: Same regarding formatting in other places/files.

**Comment by rvanheest** [comment]
Just a small thought: would it be wise to split this into 4 submodules: `easy-stage-dataset-lib`, `easy-stage-file-item-lib`, `easy-stage-dataset` (cmd) and `easy-stage-file-item` (cmd)?

**Comment by pfrolke** [comment]
@rvanheest I like this idea. I think it would make this module more clear and structured. Splitting would also allow other modules that have this module as a dependency to better define what part of this module is used.

**Comment by janvanmansum** [comment]
@rvanheest your idea is noted, but I think we should not dwell too long on this module, which is legacy soon to be phased out.

**Pull request merged**